core/src/test/java/org/pentaho/platform/util/StringUtilTest.java

```diff
@@ -75,14 +92,20 @@ public void testDoesPathContainParentPathSegment() {
     }
   }
 
-  public static void main( final String[] args ) {
-    StringUtilTest test = new StringUtilTest();
+  @Test
+  public void testMain() {
     try {
-      test.testTokenStringToArray();
-      test.testDoesPathContainParentPathSegment();
-
-    } finally {
+      Assert.assertTrue( true );
```

Comment:
Sorry - missed a line above, means to call StringUtil.main(null) to make sure it doesn't throw an exception